eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -99,7 +140,17 @@ case class Commitments(channelVersion: ChannelVersion,
 
   def addRemoteProposal(proposal: UpdateMessage): Commitments = Commitments.addRemoteProposal(this, proposal)
 
-  val announceChannel: Boolean = (channelFlags & 0x01) != 0
+  lazy val commitmentFormat: CommitmentFormat = channelVersion.commitmentFormat
+
+  lazy val localNodeId: PublicKey = localParams.nodeId
+
+  lazy val remoteNodeId: PublicKey = remoteParams.nodeId
+
+  lazy val capacity: MilliSatoshi = localCommit.spec.totalFunds
+
+  lazy val fundingTxId: ByteVector32 = commitInput.outPoint.txid
+
+  lazy val announceChannel: Boolean = (channelFlags & 0x01) != 0
```

Comment:
nit: I don't see a reason to make these `lazy val`, they will always be computed so let's just make them `val`